src/main/drivers/accgyro_mpu6050.c

```diff
@@ -176,6 +165,9 @@ static void mpu6050AccInit(void);
 static bool mpu6050AccRead(int16_t *accData);
 static void mpu6050GyroInit(void);
 static bool mpu6050GyroRead(int16_t *gyroADC);
+static void checkMPU6050DataReady(bool *mpuDataReadyPtr);
+
+static bool mpuDataReady;
```

Comment:
You should declare this variable `volatile`, it is changed asynchronously from ISR 
(or use other method to make sure that fresh value is read when neccesary)